scripts/README.md

```diff
@@ -0,0 +1,157 @@
+# Scripts
+
+This is a set of boilerplate scripts describing the [normalized script pattern that GitHub uses in its projects](https://github.blog/2015-06-30-scripts-to-rule-them-all/). The [GitHub Scripts To Rule Them All
+](https://github.com/github/scripts-to-rule-them-all) was used as a template. They were tested using Ubuntu 18.04.3 LTS on Windows 10 and using GridOS 26, a derivative of Red Hat Fedora 26, on the Lincoln Laboratory Supercomputing Cluster.
+
+- [Scripts](#scripts)
+  - [`FLOOD_ANALYSIS_CORE` and Execution](#flood_analysis_core-and-execution)
+  - [Dependencies](#dependencies)
+    - [Linux Shell](#linux-shell)
+    - [Proxy and Internet Access](#proxy-and-internet-access)
+    - [Superuser Access](#superuser-access)
+  - [The Scripts](#the-scripts)
+    - [script/bootstrap](#scriptbootstrap)
+      - [Packages](#packages)
+      - [Git Submodules](#git-submodules)
+    - [script/setup](#scriptsetup)
+      - [Data](#data)
+
+## `FLOOD_ANALYSIS_CORE` and Execution
+
+These scripts assume that `FLOOD_ANALYSIS_CORE` has been set. Refer to the repository root [README](../README.md) for instructions.
+
+## Dependencies
+
+### Linux Shell
+
+The scripts need to be run in a Linux shell. For Windows 10 users, you can use [Ubuntu on Windows](https://tutorials.ubuntu.com/tutorial/tutorial-ubuntu-on-windows#0). Specifically for Windows users, system drive and other connected drives are exposed in the `/mnt/` directory. For example, you can access the Windows C: drive via `cd /mnt/c`.
+
+If you modify these scripts, please follow the [convention guide](https://github.com/Airspace-Encounter-Models/em-overview/blob/master/CONTRIBUTING.md#convention-guide) that specifies an end of line character of `LF (\n)`. If the end of line character is changed to `CRLF (\r)`, you will get an error like this:
+
```

Comment:
The referenced error message is missing

Reply:
We used the [`em-core` convention guide](https://github.com/Airspace-Encounter-Models/em-overview/blob/master/CONTRIBUTING.md#convention-guide) as a template when drafting the language here. This is a good catch and this README should reference the [`ladi-overview` convention guide](https://github.com/LADI-Dataset/ladi-overview/blob/master/CONTRIBUTING.md)

Reply:
Resolved with commit 8f1d064